pkg/client/v1alpha2/clientset/versioned/typed/ako/v1alpha2/fake/fake_ssorule.go

```diff
@@ -36,9 +35,9 @@ type FakeSSORules struct {
 	ns   string
 }
 
-var ssorulesResource = schema.GroupVersionResource{Group: "ako.vmware.com", Version: "v1alpha2", Resource: "ssorules"}
+var ssorulesResource = v1alpha2.SchemeGroupVersion.WithResource("ssorules")
```

Comment:
Same as above

Reply:
Autogeneration is done for all crds in v1aplha2 version. So this part in L4Rule and SS0Rule is also changed.